NebulaPatcher/Patches/Dynamic/PlanetTransport_Patch.cs

```diff
@@ -35,18 +36,33 @@ public static bool SetStationStorage_Postfix(PlanetTransport __instance, int sta
         // as the game does not do that for some reason, we need to do it here
         [HarmonyPostfix]
         [HarmonyPatch("NewStationComponent")]
-        public static void NewStationComponent_Postfix(PlanetTransport __instance, StationComponent __result, int _entityId, int _pcId, PrefabDesc _desc)
+        public static void NewStationComponent_AddPlanetId_Postfix(PlanetTransport __instance, StationComponent __result, int _entityId, int _pcId, PrefabDesc _desc)
         {
             if (!SimulatedWorld.Initialized)
             {
                 return;
             }
+            
             if (!__result.isStellar && __result.planetId == 0)
             {
                 __result.planetId = __instance.planet.id;
             }
         }
 
+        [HarmonyPostfix]
+        [HarmonyPatch("NewStationComponent")]
+        public static void NewStationComponent_BroadcastNewILS_Postfix(PlanetTransport __instance, StationComponent __result, int _entityId, int _pcId, PrefabDesc _desc)
+        {
+            if (!SimulatedWorld.Initialized || !LocalPlayer.IsMasterClient) return;
```

Comment:
It feels like LocalPlayer.IsMasterClient should imply SimulatedWorld.Initialized. Nothing to fix here, just a random thought.

Reply:
I've had that thought multiple times, I'm personally just repeating what is used all over the place, similar to this.